### DISCUSSION TITLE
Email Already taken!

### DISCUSSION THREAD

**Discussion by DBJoshi** [community/community#32183] [New to GitHub] [answered]
Hello Github community! I tried to sign up using my primary email today but it gave me an error stating, "email already taken." I created the account using another email address and tried adding my primary  email address but it gave me the same error. How to resolve this issue?
Thank you.

**Comment by radar-blip** [comment]
You mean sign in? Try using just your username instead of email.

**Comment by DBJoshi** [comment]
No, I mean while registering my account.
First step is to enter your email address: that's when I got the error. So
I uses another email address. However, if I want my primary email address
on GitHub under my profile it's not allowing me to add another email
address.

**Reply by radar-blip** [reply]
What about in Settings >> Emails >> Backup email addresses
Did you try changing that to allow all ?

**Reply by DBJoshi** [reply]
Not letting me do that either! same error message!

**Comment by airtower-luna** [comment, marked as answer]
That error message sounds like someone else added that address to their account already. :thinking: 

If that's the case you should [contact GitHub support](https://support.github.com/request), hopefully they can resolve the situation. Good luck!

**Reply by 3kh0** [reply]
@DBJoshi Mark this as the answer

**Reply by nurkb23** [reply]
My bKash account

**Comment by gayathriece0612** [comment]
Ok

**Comment by nurkb23** [comment]
My add account bKash 01773696292

**Comment by nurkb23** [comment]
My bKash account add

**Comment by bretpeters3n** [comment]
I needed to swap mine just now and was having this problem. It turns out I had a Github account from years before becoming a developer from this hackathon I went to. Here is what I did to get my desired email address into my primary account.

1. use the Forgot Password feature and enter the email address in question
2. get access to that account (sign in)
3. change the email address associated with this account to another email address (not the email address in question)
4. remove the original email address from this account
5. now you can add the email address in question to your desired account

It is fairly straightforward. However, there are just enough steps involved to easily screw it up! Let me know if anyone has problems fixing their issue in this way.

**Reply by E2-Meghanadhan** [reply]
No, i too facing the same problem.. Getting Error adding me**a****gmail.com: email is already in use.. But i haven't used this in github.. 
i am trying to change my email for github account

**Reply by bretpeters3n** [reply]
Let us know if you are successful or are continuing to run into issues. Good luck :)

**Reply by rahulkishen18** [reply]
Thanks a bunch! I had the same issue with an enterprise email id and was struggling for a long time 🤣 . This finally worked!

**Reply by bretpeters3n** [reply]
Rock on 🤘

**Reply by ghost** [reply]
how was this fixed. I am facing this issue. Not able to add any new email address. Any email address i add i am getting email is already exists

**Reply by ghost** [reply]
Github support I did not get any respond too.

**Reply by raj-adarsh-1** [reply]
I have a  situation slightly different. I have a personal GitHub and a school GitHub account. Suppose I follow the above steps to add my personal GitHub to my school GitHub, it will only let me use that email address? 
Will it show my previous repos/contributions ?

**Reply by bretpeters3n** [reply]
This is a good question @Raj-Adarsh I'm not really sure about this. Definitely look into it more before changing anything.

I can tell you this though. In my situation I am no longer on the hackathon project as a contributor. However, I'm not sure if I deleted that old account or not. I must have? It does lead me to believe that the repos are not auto transferred. Sorry, this is all I can remember.

**Reply by lenikadali** [reply]
> No, i too facing the same problem.. Getting Error adding me**a****gmail.com: email is already in use.. But i haven't used this in github.. i am trying to change my email for github account

This error appears when your email address is the backup address of any other GitHub accounts you may have. In this case, go through your known GitHub accounts and find where that email is used and then remove that email address. You should be able to then use the email address as desired (fortunately GitHub reflects the changes quickly so you don't have to wait too long 😄)

The reset password option works when the primary email of the GitHub account is the same as the one you would like to swap.
